In [2]:
!pip install regex

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/


Enchant используется для проверки правописания слов и предложения исправлений для слов с ошибками. Для выполнения этой задачи он может использовать многие популярные пакеты проверки орфографии, включая ispell, aspell и MySpell. Он достаточно гибок при работе с несколькими словарями и несколькими языками.

In [1]:
pip install pyenchant

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.7/55.7 KB 912.1 kB/s eta 0:00:00


In [3]:
import re

In [7]:
# открывем файл
with open('пример_блокир.txt', 'r') as file:
    text = file.read()

# Используем регулярные выражения для поиска слов, содержащих небуквенные символы
search = re.compile(r'\b\w*[^a-zA-Z ]\w*\b') # слова, которые начинаются с нуля или более букв или цифр, за которыми следует любой символ, кроме буквы или пробела, за которым следует ноль или более букв или цифр.
words = search.findall(text)
# слова, которые написаны правильно, но содержат специальные символы
misspelled_words = []
for match in words: # проходим по каждому слову из списка words
    if not re.match(r'^\w+$', match): # смотрим на соответствие ^\w+$ - строка, которая состоит только из буквенно-цифровых символов без пробелов иди др. символов
        misspelled_words.append(match)
#печать
print("Misspelled words: ", misspelled_words)


Misspelled words:  ['double-J', 'v@x', 'Tw!tter', 'can’t', 'child’s', 'Gun-bullet', 'town/city', 'unv@xx', 'v@xx', 'v@xx', 'v@xx', '8m!ce', 'Share\nV', 'doesn’t', 'doesn’t', 'j@bb', 'v@xx', 'did.That', '’s', 'vein/artery', '.Either', 'Ph@rma', 'week\nDirect', 'air\nManaging', 'full\nAdequate', 'sleeps\nWalking', '/exercise', 'Let’s', 'j@bb', 'didn’t', 'wasn’t', 'C0VID\nRemd3sivir', 'P@X', 'Real-life', 'v@cc', '!nes', 'theorist”for', 'cl0ts\nCl0ts', 'str0kes\nCl0ts', 'emb0lism/PE', '\nCl0ts', 'miscarr!ages', '\nSh0ts', 'dis0rders\nSh0ts', 'canc3r\nSh0ts', 's@fe', 'S@RS', 'There’s', 'S@RS', 'doesn’t', '0.1', '0.125', '0.06', '0.100', '0.300', 'Natto’s', 'MK-7', 'j@bbed', 'supplement/food', 'you’re', 'p@xl0vid', 'don’t', 'P@xl0vid', 'life-threatening', 'Jeremy’s', 'don’t', 'don’t', 'pf!z3r', 'document\nSpecifically', 'de@d', 'ph@rma', 'child’s', 'It’s', 'v@xx', 'G@tes', '’s', 'v@cc', '!nes', 'Fos@max', '24/7', 'I’m', 'v-a', '-x']


In [8]:
import enchant

In [11]:
# Загрузим словарь
d = enchant.Dict("en_US")

# откроем текст
with open("пример_блокир.txt", "r", encoding='utf-8') as open_file: 
    text = open_file.read()
    
words = text.split()# разделим 

# Проверка слов
misspelled_words = []
for word in words:
    if not d.check(word): #check проверяет слова
        misspelled_words.append(word)

# вывод слов
print("Misspelled words: ", misspelled_words)

Misspelled words:  ['double-J@', 'bbed,', 'C0VID…', '…', 'v@x', 'Tw!tter', '“Misinf0rmation”', 'GETTR.', '“complications', 'stroke”….2', 'B00ST.', 'Facts:', 'B00ST3R', 'pfiz3r', 'j@', 'bbs', 'transmissi0n', '“move', '&', 'risk”', 'inclu', '&', 'IV3RM3CTIN,', 'Hydr0xychl0r0,', 'EGCG,', 'QUERCETIN;', 'asking…', 'town/city', 'unvaccinated', 'b00sters.', 'gl0balist', 'unv@xx.', 'ICD', 'illness,', 'C0VID', 'v@xx', 'year,', '&', 'j@', 'bbs', '“Emergency', 'Authorization”', '“emergency“', 'v@xx', 'v@xx', '8m!ce', '“friends”', 'j@', 'bb', 'B00ST3R', '“selfish”….you', 'V@', 'g3netic', 'v@', 'damage,', 'damage,', 'McCull0ugh', 'j@bb', 'ur', 'v@xx', 'did.That’s', 'ur', 'vein/artery.Either', 'target:', 'brain,', 'heart,', 'gland,', '&', 'Ph@rma:', '&', 'Walking/exercise', 'here:', 'C0VID', "'matches", 'M0derna', '“THREE', 'YEARS”', "began'", '3trillion', 'C0VID', 'j@bb.', 'Folks,', 'weeks,', 'ur', 'lymphoma;', 'B00ST.', 'Remd3sivir', '&', 'syncytial', '(RSV),', '“repurposed”', '&Marburg;', '“repur

In [12]:
#видим, что вывадся & и ... 
text = re.sub(r'[&…]', '', text) #Sub возвращает новую строку, полученную в результате замены по указанному шаблону

In [13]:
# вывод слов с частотностью 
misspelled_words = {}
for word in words:
    if not d.check(word): # check проверяет слово 
        if word in misspelled_words:
            misspelled_words[word] += 1
        else:
            misspelled_words[word] = 1

# вывод + частотность ошибок
print("Misspelled words: ")
for word, frequency in misspelled_words.items():
    print(word, ":", frequency)

Misspelled words: 
double-J@ : 1
bbed, : 1
C0VID… : 1
… : 1
v@x : 1
Tw!tter : 1
“Misinf0rmation” : 1
GETTR. : 1
“complications : 1
stroke”….2 : 1
B00ST. : 2
Facts: : 1
B00ST3R : 2
pfiz3r : 1
j@ : 6
bbs : 2
transmissi0n : 1
“move : 1
& : 8
risk” : 1
inclu : 1
IV3RM3CTIN, : 1
Hydr0xychl0r0, : 1
EGCG, : 1
QUERCETIN; : 1
asking… : 1
town/city : 1
unvaccinated : 1
b00sters. : 1
gl0balist : 1
unv@xx. : 1
ICD : 1
illness, : 1
C0VID : 8
v@xx : 4
year, : 1
“Emergency : 1
Authorization” : 1
“emergency“ : 1
8m!ce : 1
“friends” : 1
bb : 3
“selfish”….you : 1
V@ : 1
g3netic : 1
v@ : 1
damage, : 2
McCull0ugh : 1
j@bb : 1
ur : 3
did.That’s : 1
vein/artery.Either : 1
target: : 1
brain, : 3
heart, : 1
gland, : 1
Ph@rma: : 1
Walking/exercise : 1
here: : 1
'matches : 1
M0derna : 1
“THREE : 1
YEARS” : 1
began' : 1
3trillion : 1
j@bb. : 1
Folks, : 1
weeks, : 1
lymphoma; : 1
Remd3sivir : 2
syncytial : 1
(RSV), : 1
“repurposed” : 2
&Marburg; : 1
P@X, : 1
C0VID. : 1
patients, : 1
disorders, : 1
injury, : 1
ser

In [ ]:
# поробуем исправить слова 
for word in words:
  if not d.check(word):
    suggestions = d.suggest(word) # Мы можем вывести список возможных исправлений слова c d.suggest
    print (f"Misspeled word: {word}. Suggestions:{suggestions}")

Misspeled word: double-J@. Suggestions:['double-J', 'double-Jr', 'double-Jo', 'double-JD', 'double-JP', 'double-JV', 'double-Jg', 'doublet']
Misspeled word: bbed,. Suggestions:['bobbed']
Misspeled word: C0VID. Suggestions:['VIVID']
Misspeled word: v@x. Suggestions:['vex', 'VAX']
Misspeled word: Tw!tter. Suggestions:['Twitter']
Misspeled word: “Misinf0rmation”. Suggestions:['misinformation']
Misspeled word: GETTR.. Suggestions:['GET TR', 'GET-TR', 'GETTY']
Misspeled word: “complications. Suggestions:['complications', 'complication', 'simplification', 'misapplication', 'compilations']
Misspeled word: stroke”.2. Suggestions:['stroke']
Misspeled word: B00ST.. Suggestions:[]
Misspeled word: B00ST3R. Suggestions:[]
Misspeled word: pfiz3r. Suggestions:[]
Misspeled word: j@. Suggestions:['j', 'jr', 'jg']
Misspeled word: bbs. Suggestions:['BBS', 'ebbs', 'bibs', 'bobs', 'bubs', 'bis', 'abs', 'tbs', 'obs', 'bbl', 'lbs', 'bus', 'bps']
Misspeled word: transmissi0n. Suggestions:['transmission', 'tra

In [ ]:
# мы можем увидеть, что 0 можно заменить на о, @ на a, 3 на e

In [ ]:
# заменим их на буквы
patterns = {
  
    "@": "a",
    "0": "o",
    "!": "i",
    "3": "e",
    "IV3RM3CTIN": "ivermectin"
}
for pattern, replacement in patterns.items():
    text = re.sub(pattern, replacement, text)

print(text)

UK government has admitted that once you have been double-Ja bbed, you will never again be able to acquire full natural immunity to CoVID or possibly any other virus  now Insurance companies are refusing to pay out a life insurance policy after the insured died from the vax

Twitter would not let me add the link. “Misinformation” so link in GETTR. Same handle.

Just lost another 9 year old due to “complications from a stroke”.2 weeks after a BooST. Parent you really have one important job and that is protecting children from these monsters. You just can’t fail at that.

Know the Facts

Before next BooSTeR know that pfizer never tested if ja bbs would prevent transmission because they had to “move at the speed of science to understand what is happening in the market  we had to do everything at risk”
Risk inclu you  your child’s life

IVeRMeCTIN, Hydroxychloro, EGCG, QUERCETIN; are other great transporter of zinc.  Once zinc is in the cell it does a fantastic job of inhibiting the virus 

 language_tool_python - проверяет грамматику


In [ ]:
!pip install -q transformers
from transformers import pipeline
sentiment_pipeline = pipeline("sentiment-analysis")
data = ["UK government has admitted that once you have been double-Ja bbed, you will never again be able to acquire full natural immunity to CoVID or possibly any other virus  now Insurance companies are refusing to pay out a life insurance policy after the insured died from the vax"]
sentiment_pipeline(data)

No model was supplied, defaulted to distilbert-base-uncased-finetuned-sst-2-english and revision af0f99b (https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english).
Using a pipeline without specifying a model name and revision in production is not recommended.


[{'label': 'NEGATIVE', 'score': 0.9770331382751465}]

In [ ]:
import language_tool_python


UK government has admitted that once you have been double-J@ bbed, you will never again be able to acquire full natural immunity to C0VID… or possibly any other virus … now Insurance companies are refusing to pay out a life insurance policy after the insured died from the v@x

Tw!tter would not let me add the link. “Misinf0rmation” so link in GETTR. Same handle.

Just lost another 9 year old due to “complications from a stroke”….2 weeks after a B00ST. Parent you really have one important job and that is protecting children from these monsters. You just can’t fail at that.

Know the Facts:

Before next B00ST3R know that pfiz3r never tested if j@ bbs would prevent transmissi0n because they had to “move at the speed of science to understand what is happening in the market & we had to do everything at risk”
Risk inclu you & your child’s life

IV3RM3CTIN, Hydr0xychl0r0, EGCG, QUERCETIN; are other great transporter of zinc.  Once zinc is in the cell it does a fantastic job of inhibiting the virus from replicating. Gun-bullet analogy described by some.  Zinc is the bullet.

I remember Bill saying asking… would it be great if there was an outbreak in a particular town/city that government could look at the data and see exactly how many people are unvaccinated or behind on their b00sters.  
No good can come from this data in a gl0balist hands.

Federal Government Is Tracking the unv@xx. New ICD codes are not based on disease or illness, but are based on C0VID v@xx status. It took almost a year, but in January 2023 these codes became available to nearly every medical clinic & hospital in the USA

These j@ bbs are still under “Emergency Use Authorization”  

This is further documentation that be “emergency“ was never intended to go away. I believe the first v@xx ever be added to the regular v@xx series while still under emergency authorization label.

8m!ce tested however.

If you lost a few “friends” or family members because you refused to take the j@ bb or B00ST3R and you were judged as being “selfish”….you really lost nothing and are certainly better for it.
RT and Share
V@ xx doesn’t leave the body. The g3netic material doesn’t  break down.  
FDA says the v@ xx causes heart damage, brain damage, and blood clots.

P McCull0ugh 

The j@bb was never intended to stay in ur arm as other v@xx did.That’s why they did not advocate for pulling back on the plunger as it made no difference if it got into ur vein/artery.Either way it would get to the intended target: brain, heart, adrenal gland, ovaries & testicles

Healthy options that you cannot get from a pill or from Big Ph@rma:

Lifting heavy weights 3 to 4 times a week
Direct sunlight & fresh air
Managing your stress 
Breathing deep and full
Adequate sleeps
Walking/exercise

Nothing to see here:
C0VID virus contains tiny chunk of DNA that 'matches sequence patented by M0derna “THREE YEARS” before pandemic began'

A one in 3trillion chance C0VID developed the code naturally.

Let’s just all disregard this and move along.

Yesterday I heard that there are companies now docking peoples pay for not taking the j@bb. We are not out of the woods.

Folks, if you have a new lump in your neck or armpit that is therefore more than 3 to 4 weeks, please get it evaluated. Even if ur doctor tells you that is normal get a 2nd or 3rd opinion. So many people now presenting with advanced lymphoma; particularly after the first B00ST.

Remd3sivir was developed in 2009 by Gilead Sciences to treat hepatitis C & respiratory syncytial virus (RSV), but it didn’t work against those. So they “repurposed” it for Ebola &Marburg; wasn’t effective in stopping replication either. So they once again “repurposed” for C0VID
Remd3sivir and P@X, both new and untested for C0VID.  

Real-life data from more than 5000 C0VID patients, support that kidney disorders, almost exclusively acute kidney injury, represent a serious, early, and potentially fatal adverse drug reaction of remd3sivir.

Remember when they told you that the j@ bb stays in your arm “ just like other v@cc!nes.  Now folks are finding it in the their blood, brain, adrenals, testicles, b0ne marrow and 0varies.  Yep….they lied!!!

Nighttime heartburn affects 4 out of 5 ppl who suffer regular heartburn &acid reflux. Prevent heartburn by limiting acidic foods, such as grapefruit, and tomatoes.  Also cut back on pepper or chilies especially later in the day

For about 3 yrs now ppl were labeled as “conspiracy theorist”for saying what we see at the ground level that;
Sh0ts cause cl0ts
Cl0ts cause str0kes
Cl0ts cause pulmonary emb0lism/PE
Cl0ts cause miscarr!ages
Sh0ts cause aut0immune dis0rders
Sh0ts cause canc3r
Sh0ts NOT s@fe

Know this:

Natural !mmunity from S@RS C0V 1 lasts upwards of 20 years. There’s absolutely no reason to think that natural !mmunity from the patented newer version, S@RS C0V 2, doesn’t offer the same natural protection.

C0r0navirus particles are 0.1 micron.

C0VID particle size is 0.125 microns; range is 0.06 microns to .14 microns. Pore size in N95 masks is~ 0.100 to 0.300 micron.

Standard surgical mask….material of the mask to filter particles (mean diameter close to 3 micron).

Posted about natto a few weeks ago. Natto has far more Vit K2 than any other food….also about 90% of Natto’s K2 is in MK-7 form If you are at risk for cl0ts,(ppl who have been recently j@bbed), should really consider adding this to their supplement/food unless contraindicated

The only reason that you’re getting “free C0VID homeKit” from the government is to give you p@xl0vid even when your symptoms are “mild” per CDC. Please don’t fall for it.
Black box warning has been given to Rit0navir (included in P@xl0vid treatment) with side effects include life-threatening liver, pancreas and heart issues. 

Jeremy’s drug history does not negate the known complications of the B00sted to include sudden cardiac death.



Does it seem like only the common folks are dropping dead from a “medical emergency.” Why is it that you don’t see any of the gl0balists or their families having this “sudden cardiac death.” Maybe they know something we don’t?

Know the facts: 
See pf!z3r document
Specifically 1223 de@d in 3 months of testing page 7 and then see the last 9 pages. Many of the possible outcomes are autoimmune disorders.

Still “safe and effective”? 

Before allowing your child to get j@ bbed please do your own research. The “3 lettered agencies” have been corrupted &now pay homage to big ph@rma. Your child’s future falls squarely in your hands. It’s not about what friends& neighbors are doing. This is your child, protect them

If your supportive evidence regarding anything C0VID /v@xx is from G@tes’s three  lettered agencies: FDA, CDC, NIH, or WH0 then you are clearly missing the point.

Remember when they told you that the j@ bb stays in your arm “ just like other v@cc!nes.  Now folks are finding it in the their blood, brain, adrenals, testicles, bone marrow and ovaries.  Yep….they lied!!!
Osteoporosis drugs such as Act0nel, B0niva, & Fos@max may be assoc with a doubling in esophageal cancer risk. Could “lower the amount of calcium” in your blood. Fosamax and other bisphosphonates …. "you may be at risk of developing osteonecrosis (bone death) of the jaw.
RT and Share

‘I got c0vid!’-No stupid, you took a fake test with fake positives. You got the cold or flu like you always did. ‘But, it was worse this time, it was different, it WAS c0vid!’-No, stupid, your perception was altered by a 24/7 MSM fear campaign & brainwashing.

My story. I’m going for 22 months and still many issues post v-a-x:
After nearly Seven months I am still dealing with side effects from both doses of the C0vid_vaccine_ (M0derna), which I received on March 17 then June 17, 2021:
